src/sb_pointcloud_processing/src/IGVCVisualizerNode.cpp

```diff
@@ -0,0 +1,152 @@
+//
+// Created by robyn on 08/03/18.
+//
+
+#include "IGVCVisualizerNode.h"
+
+
+// Allocation of memory for static member variables
+float IGVCVisualizerNode::h_margin_of_error;
```

Comment:
Are these definitions necessary?

Reply:
I was getting an error before for undefined reference to static member variables. So i used this to fix it. https://stackoverflow.com/questions/9110487/undefined-reference-to-a-static-member